js/main.js

```diff
@@ -1,6 +1,139 @@
+    technologies: ['HTML', 'Bootstrap', 'Ruby'],
+    mProject: false,
+  },
+};
+
+function loadWorks() {
+  let topCard = '';
+  let cardGroup = '';
+  //  eslint-disable-next-line
+  for(const project in projects) {
+    const proj = projects[project];
+    let technologies = '';
+    const projTech = proj.technologies;
+    if (!proj.mProject) {
+      for (let i = 0; i < projTech.length; i += 1) {
+        technologies += `\t\t\t\t\t\t<button class="p-lang-2">${projTech[i]}</button>\n`;
+      }
+      cardGroup += `
+      <div id="${project}" class="card column wht bg-img-1" onmouseover="mauseOn('${project}')" onmouseout="mauseOff('${project}')">\n
+      <div class="card-content">\n
+        <h2 class="title-post-2">${proj.title}</h2>\n
+        <p class="tema-2">${proj.description}</p>\n
+        <ul class="btn-group">\n
+        ${technologies}
+        </ul>\n
+      </div>\n
+      <button class="link-btn-2 fill-h" onclick="seeProject('${project}')">See Project</button>\n
+    </div>\n`;
+    } else {
+      for (let i = 0; i < projTech.length; i += 1) {
+        technologies += `\t\t\t\t\t\t<button class="p-lang">${projTech[i]}</button>\n`;
+      }
+      topCard += `
+      \t\t\t\t<div class="fit-container g-1">\n
+      \t\t\t\t\t<img class="fit-container" src="${proj.imgb}" alt="picture">\n
+      \t\t\t\t</div>\n
+      \t\t\t\t<div class="fit-container g-1">\n
+      \t\t\t\t\t<h2 class="title-post">${proj.title}</h2>\n
+      \t\t\t\t\t<p class="tema-2">${proj.description}</p>\n
+      \t\t\t\t\t<div class="btn-group g-1 spaced">\n
+      ${technologies}
+      \t\t\t\t\t</div>\n
+      \t\t\t\t\t<button class="link-btn fit" onclick="seeProject('${project}')">See Project</button>\n
+      \t\t\t\t</div>\n\n`;
+    }
+  }
+  document.getElementById('top-card').innerHTML = topCard;
+  document.getElementById('card-group').innerHTML = cardGroup;
+}
+
 function displayMenu() {
   document.querySelector('#menu-button').classList.toggle('hidden');
   document.querySelector('#wide-menu').classList.toggle('hidden');
+  document.querySelector('body').classList.toggle('noscroll');
+}
+
+// eslint-disable-next-line
+function seeProject(id) {
+  const project = projects[id];
+  let technologies = '';
+  for (let i = 0; i < project.technologies.length; i += 1) {
+    technologies += `\t\t\t\t\t\t<button class="p-lang g-1">${project.technologies[i]}</button>`;
+  }
+  document.querySelector('#card-modal').classList.toggle('hidden');
+  document.querySelector('#modal-title').textContent = project.title;
+  document.getElementById('modal-tecs').innerHTML = technologies;
+  document.querySelector('#modal-img').textContent = project.imgb;
+  document.querySelector('#modal-desc').textContent = project.description;
+}
+
+function closeModal() {
+  document.querySelector('#card-modal').classList.toggle('hidden');
```

Comment:
- Hey :hand: Here you did a great job here but make sure your project working properly as explained below when we clique to the see project button shown below  the popup appear with the detail of the project :+1: :smiley: 


![image](https://user-images.githubusercontent.com/95297251/190876467-539a8a75-bb60-4cb3-b54d-e22af2ab886b.png)

The [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters is set correctly project repository.
When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.
The popup is responsive, it fits mobile and desktop sizes properly.
The pop-up follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).